code/baselineTiming.py

```diff
@@ -51,7 +51,7 @@ def main():
     print "\n -------------- Read MERG Data ----------"
     print "\n Start the timer for the data ingest process"
     readMergStart = time.time()
-    mergImgs, timeList, LAT, LON, userVariables = iomethods.read_data('ch4', 'latitude', 'longitude', userVariables)
+    mergImgs, timeList, LAT, LON, userVariables = iomethods.read_data('ch4', 'latitude', 'longitude', userVariables, fileType='netCDF')
```

Comment:
read_data accepts 4 parameters. I think I see the logic of this parameter though. A cleaner way might be to get it from the user in the config.json file, then as u are passing userVariables already, you'll get that variable as well.